griptape/tools/google_drive/tool.py

```diff
@@ -244,7 +250,7 @@ def search_files(self, params: dict) -> ListArtifact | ErrorArtifact:
             logging.error("MalformedError occurred")
             return ErrorArtifact("error searching for file due to malformed credentials")
 
-    def _build_client(self, scopes: list[str]) -> Resource:
+    def _build_client(self, scopes: list[str]) -> Any:
```

Comment:
`Any`?

Reply:
We can't use the `Resource` type because it comes from google. And the `Resource` type is dynamically generated anyways so there aren't really useful type hints in it.